letsencrypt/README.rst

```diff
@@ -97,6 +114,13 @@ an upstream for your odoo instance and do something like::
         proxy_pass    http://yourodooupstream;
     }
 
+If you're using a multi-database installation (with or without dbfilter option)
```

Comment:
> in the original version, you have one key/certificate per database, because the letsencrypt-folder is a sibling of the filestore folder

In the original version, default data_dir was used (.local/share/Odoo) instead of configured data_dir. So, there was only one letsencrypt folder for each Odoo instance, even with dbfilter actived and filtering databases (SaaS use case).

So, why this approach is not working in SaaS use case?

> As soon as you get serious with the website module, you'll need one domain per database anyways

You're right, but this addon is not only for website scenarios, but for traditional ERP backend scenarios as well. With this implementation you have both scenarios covered.